test/Main.purs

```diff
@@ -55,42 +56,44 @@ shouldFail ∷ String → Aff Unit
 shouldFail input =
   case parseInsect initialEnvironment input of
    Left _ → pure unit
-   Right _ → failure $ "input is expected to throw a parse error: '" <> input <> "'"
+   Right _ → fail $ "input is expected to throw a parse error: '" <> input <> "'"
 
 expectOutput ∷ Environment → String → String → Aff Unit
 expectOutput env expected inp =
   let { msg: out } = repl fmtPlain env inp
   in
     unless (out == expected) do
-      failure $ "Unexpected result:\n" <>
-                "Input:    '" <> inp <> "'\n" <>
-                "Output:   '" <> out <> "'\n" <>
-                "Expected: '" <> expected <> "'\n"
+      fail $ "Unexpected result:\n" <>
+             "Input:    '" <> inp <> "'\n" <>
+             "Output:   '" <> out <> "'\n" <>
+             "Expected: '" <> expected <> "'\n"
 
 prettyPrintCheck ∷ String → Aff Unit
 prettyPrintCheck input =
   case parseInsect initialEnvironment input of
     Left err →
       case parseErrorPosition err of
         Position pos →
-          failure $ "Parse error for input '" <> input <> "': "
+          fail $ "Parse error for input '" <> input <> "': "
                                 <> parseErrorMessage err
                                 <> " at position "
                                 <> show pos.column
     Right output@(Expression expr) →
       shouldParseAs output (format fmtPlain (pretty expr))
-    _ → failure "Input is not an expression"
-
+    _ → fail "Input is not an expression"
 
 main ∷ Effect Unit
-main = runTest do
+main = launchAff_ $ runSpec [specReporter] do
```

Comment:
There's also `consoleReporter`, which I think looks a little prettier, but `specReporter` shows tests that took a longer time to run and how long they took, which is also nice. I'm not sure how useful that would be though.

`specReporter`:

![`specReporter`](https://github.com/sharkdp/insect/assets/32234660/c7998654-7a50-4c4f-a9ca-707eb8afc47a)

`consoleReporter`:

![`consoleReporter`](https://github.com/sharkdp/insect/assets/32234660/a0bca245-7968-4850-8d84-8ce125ca6888)

Opinions?

Reply:
I went ahead and chose `consoleReporter`.